### ISSUE TITLE
About Train In ImageNet

### ISSUE THREAD

**Issue by ylc2580** [miaow1988/ShuffleNet_V2_pytorch_caffe#25]
hi,
   I use your code with pytorch to train on imagenet. At the begin,the loss is very small,it is around 0.02.the speed of loss decrease is around 0.0001. do it ok?
   thank you very much!

**Comment by miaow1988** [comment]
Normally the initial loss of ImageNet should be around 6.9.
